src/flamenco/runtime/sysvar/fd_sysvar_epoch_rewards.c

```diff
@@ -88,10 +88,18 @@ fd_sysvar_epoch_rewards_init(
     FD_TEST( total_rewards >= distributed_rewards );
 
     fd_sysvar_epoch_rewards_t epoch_rewards = {
-        .epoch_rewards={
-            .distributed_rewards = distributed_rewards,
-            .total_rewards = total_rewards,
-            .distribution_complete_block_height = distribution_complete_block_height
+      // THIS IS ALL WRONG... this needs to be fixed for how epoch rewards have been rewritten
```

Comment:
Could you add a comment at the top of this file clarifying that this is still potentially a WIP and so subject to change in Agave, and that this account does not exist in mainnet or testnet yet?